sample-app/src/pages/VerticalSearchPage.tsx

```diff
@@ -11,7 +11,8 @@ import '../sass/VerticalSearchPage.scss';
 import { StandardCard } from '../components/cards/StandardCard';
 import { useLayoutEffect } from 'react';
 import { useAnswersActions } from '@yext/answers-headless-react';
-import { SearchIntent } from '@yext/answers-headless-react';
+import FilterSearch from '../components/FilterSearch';
+import { SearchIntent } from '@yext/answers-headless';
```

Comment:
let's keep the imports to be from `@yext/answers-headless-react`

Reply:
fixed